rasa/core/policies/memoization.py

```diff
@@ -239,20 +251,52 @@ def predict_action_probabilities(
         return self._prediction(result)
 
     def _metadata(self) -> Dict[Text, Any]:
-        return {
-            "priority": self.priority,
-            "max_history": self.max_history,
-            "lookup": self.lookup,
-        }
+        return {"lookup": self.lookup}
 
     @classmethod
     def _metadata_filename(cls) -> Text:
         return "memorized_turns.json"
 
+    @classmethod
+    def load(
+        cls,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+        **kwargs: Any,
+    ) -> "MemoizationPolicyGraphComponent":
+        """Loads a trained policy (see parent class for full docstring)."""
+        featurizer = None
+        lookup = None
+
+        try:
+            with model_storage.read_from(resource) as path:
+                metadata_file = Path(path) / cls._metadata_filename()
+                metadata = json.loads(rasa.shared.utils.io.read_file(metadata_file))
+                lookup = metadata["lookup"]
+
+                if (Path(path) / FEATURIZER_FILE).is_file():
+                    featurizer = TrackerFeaturizer.load(path)
+
+        except (ValueError, FileNotFoundError, FileIOException):
+            logger.info(
```

Comment:
`TEDPolicy` and other policies do a `warning`. How about changing this here, too, to make it consistent?
```suggestion
            logger.warning(
```